common/script_values/wc_magic_lifestyles_values.txt

```diff
@@ -0,0 +1,545 @@
+			}
+		}
+		add = 1989
+	}
+}
+
+wc_magic_lifestyle_auto_selection_weight_education_intrigue_add_to_base_value = 200
+wc_magic_lifestyle_auto_selection_weight_education_martial_add_to_base_value = 500
+wc_magic_lifestyle_auto_selection_weight_education_learning_add_to_base_value = 1000
+
+@wc_local_lifestyle_multiply_mod_positive_big = 10
+@wc_local_lifestyle_multiply_mod_positive_medium = 5
+@wc_local_lifestyle_multiply_mod_positive_small = 3
+@wc_local_lifestyle_multiply_mod_positive_tiny = 1.5
+
+@wc_local_lifestyle_multiply_mod_negative_medium = 0.25
+
+wc_magic_lifestyle_focus_selection_weight_multiply_all_magic_value = { # Common multiplier of all focuses
+	value = 1
+
+	if = {
+		limit = {
+			OR = {
+				has_trait = magic_bad
+				has_trait = magic_bad_noninheritable
+			}
+		}
+		multiply = 0
+	}
+}
+wc_magic_lifestyle_focus_selection_weight_multiply_elemental_spirit_magic_value = {
+	value = 1
+
+	if = {
+		limit = { MAGIC_is_approved_trigger = { MAGIC = elemental_spirit } }
+		multiply = @[wc_local_lifestyle_multiply_mod_positive_big]
+	}
+
+	if = {
+		limit = {
+			MAGIC_is_shunned_or_illegal_trigger = { MAGIC = elemental_spirit }
+			NOT = { has_trait = cynical }
+		}
+		multiply = 0
+	}
+
+	if = { # Elemental duality - bravery
+		limit = { has_trait = brave }
+		multiply = @[wc_local_lifestyle_multiply_mod_positive_medium]
+	}
+	else_if = {
+		limit = { has_trait = craven }
+		multiply = @[wc_local_lifestyle_multiply_mod_negative_medium]
+	}
+	if = { # Elemental duality - naivete
+		limit = { has_trait = trusting }
+		multiply = @[wc_local_lifestyle_multiply_mod_positive_medium]
+	}
+	else_if = {
+		limit = { has_trait = paranoid }
+		multiply = @[wc_local_lifestyle_multiply_mod_negative_medium]
+	}
+}
+wc_magic_lifestyle_focus_selection_weight_multiply_elemental_decay_magic_value = {
+	value = 1
+
+	if = {
+		limit = { MAGIC_is_approved_trigger = { MAGIC = elemental_decay } }
+		multiply = @[wc_local_lifestyle_multiply_mod_positive_big]
+	}
+
+	if = {
+		limit = {
+			MAGIC_is_shunned_or_illegal_trigger = { MAGIC = elemental_decay }
+			NOT = { has_trait = cynical }
+		}
+		multiply = 0
+	}
+
+	if = { # Elemental duality - efficiency
+		limit = { has_trait = diligent }
+		multiply = @[wc_local_lifestyle_multiply_mod_positive_medium]
+	}
+	else_if = {
+		limit = { has_trait = lazy }
+		multiply = @[wc_local_lifestyle_multiply_mod_negative_medium]
+	}
+	if = { # Elemental duality - ruthlessness
+		limit = { has_trait = callous }
+		multiply = @[wc_local_lifestyle_multiply_mod_positive_medium]
+	}
+	else_if = {
+		limit = { has_trait = sadistic } # Cruelty for cruelty's sake is not as efficient
+		multiply = @[wc_local_lifestyle_multiply_mod_positive_small]
+	}
+	else_if = {
+		limit = { has_trait = compassionate }
+		multiply = @[wc_local_lifestyle_multiply_mod_negative_medium]
+	}
+}
+
+wc_approved_magic_auto_selection_multiply_value = { # Cleargy and zealous prefer approved magic stronger than others
+	value = 1
```

Comment:
Shouldn't "normal" people also prefer approved magic? Maybe not by as much as those demographics, but at least by a little bit.

Reply:
Nevermind, they already do